arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/await/AwaitScope.kt

```diff
@@ -0,0 +1,72 @@
+package arrow.fx.coroutines.await
+
+import arrow.atomic.Atomic
+import arrow.atomic.update
+import kotlinx.coroutines.async as coroutinesAsync
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.CoroutineStart
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.coroutineScope
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+public suspend fun <A> await(
+  block: suspend AwaitScope.() -> A
```

Comment:
```suggestion
public suspend fun <A> awaitAll(
  block: suspend AwaitAllScope.() -> A
```

Would this naming make more sense in relation to the KotlinX Coroutines naming?

Reply:
Done in the new commit